lib/models/listings/furniture_listing.js

```diff
@@ -6,7 +6,18 @@ var secure = require('./shared/secure.js');
 var base_listing = require('./listing.js');
 var photo = require('./shared/photo.js');
 var photoSchema = photo.schema;
+var validation = require('../../validation.js');
 
+var FURNITURE_TITLE_LEN_MIN = 1;
```

Comment:
Sweet
> On Sep 10, 2016, at 1:46 PM, Brian Cefali <notifications@github.com> wrote:
> 
> In lib/models/listings/furniture_listing.js <https://github.com/TuftsTusk/ringmaster/pull/19#discussion_r78281062>:
> 
> >  
> > +var FURNITURE_TITLE_LEN_MIN = 1;
> Yep, good catch. I'm making the inequalities inclusive right now.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/TuftsTusk/ringmaster/pull/19/files/96964e88d3de90e817d95defd3499cfb700997ec#r78281062>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ACtOnROqYI7bT_eF7JolEvcq8ToZhCl_ks5qou0PgaJpZM4J4nyW>.
>